src/ui/components/Icon/img/external-black.svg

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I had to create this because the icon we currently have for this, called `external`, is blue, and we want this to be black.